scripts/document_types/publication.py

```diff
@@ -190,8 +196,12 @@ def get_publication_data(connection, limit=0):
                         full_pvalue = True
                     publication_document['fullPvalueSet'] = full_pvalue
 
-               
-               
+                ##########################################3
+                # Get a list of countries of recruitment
+                ##########################################3
+                df = pd.read_sql(country_of_recruitment_sql, connection, params = {'publication_id' : publication[0]})
```

Comment:
In our team meeting, we decided not to use Pandas.

Reply:
When was that meeting? The whole variant.py module depends on pandas. Also, what was the argument behind that decision?